packages/playground/src/components/layout/layout-header/header-account/header-account.tsx

```diff
@@ -75,6 +75,9 @@ export class HeaderAccount {
       await window["ethereum"].enable();
     } catch {
       console.error("Was not able to call `window.ethereum.enable()`");
+      window.alert(
+        "Your browser does not support enabling your wallet programatically. Please unlock your Web3 wallet and try again."
```

Comment:
`window.alert`?

Reply:
Yeah it's a massive edge case so don't want to spend too much time on this with proper UI etc